records/generator.go

```diff
@@ -25,6 +26,8 @@ import (
 // Map host/service name to DNS answer
 // REFACTOR - when discoveryinfo is integrated
 // Will likely become map[string][]discoveryinfo
+// Effectively we're (ab)using the map type as a set
+// It used to have the type: rrs map[string][]string
```

Comment:
i think you're right, important during the "build" phase (trying to avoid duplicate entries)